### PR TITLE
Sync template v1.10.1

### PR DESCRIPTION
**Pull request by phue**
# nf-core/methylseq pull request

Many thanks for contributing to nf-core/methylseq!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/methylseq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/methylseq)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/methylseq/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ewels** [comment]
Note that there was a bug in one of the github actions files 😢 

Fix: https://github.com/nf-core/tools/pull/713/files#diff-0908bd37f0e6c722adde926083d516df (mostly whitespace, just missing `job:`)

New patch release coming out to fix this tomorrow morning hopefully (but that template merge will be _very_ easy if you've already done this 👍🏻 )

Great if this update finishes and you can find any other bugs before then! 😅 

Phil

**Comment by phue** [comment]
> Great if this update finishes and you can find any other bugs before then! sweat_smile

I'll try, the only mini thing I noticed so far is that the `full_test.config` still uses `readPaths` instead of `input_paths`, so you might want to squeeze that in:

https://github.com/nf-core/tools/pull/715

https://github.com/nf-core/tools/blob/2ccb064aa90f1df3065af2faffc0c21b2d54cdc5/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/conf/test_full.config#L18

File: main.nf [hunk 1]
```diff
@@ -330,7 +339,7 @@ Channel.from(summary.collect{ [it.key, it.value] })
  * Parse software version numbers
  */
 process get_software_versions {
-    publishDir "${params.outdir}/pipeline_info", mode: 'copy',
+    publishDir "${params.outdir}/pipeline_info", mode: params.publish_dir_mode,
```
Comment (ewels) [inline main.nf:342]: I guess we need to make this change for all other processes too - I'll do that in a separate PR to check some other stuff now though 👍🏻

File: .github/workflows/push_dockerhub.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# This builds the docker image and pushes it to DockerHub
+# Runs on nf-core repo releases and push event to 'dev' branch (PR merges)
+on:
+  push:
+    branches:
+      - dev
+  release:
+    types: [published]
+
+push_dockerhub:
```
Comment (ewels) [inline .github/workflows/push_dockerhub.yml:11]: This is the line that needs the extra `job:` - will also fix this in a second.

**Pull request merged**